src/libs/l2ps/zk/L2PSBatchProver.ts

```
@@ -0,0 +1,584 @@
+            });
+
+            this.childProcess.stdin?.write(request);
+        });
+    }
+
+    /**
+     * Terminate subprocess
+     */
+    async terminate(): Promise<void> {
+        if (this.childProcess) {
+            this.childProcess.kill();
```

ESLint

[error] Extra semicolon.

(semi)